content/en/docs/reference/command-line-tools-reference/kube-apiserver.md

```diff
@@ -344,7 +344,7 @@ kube-apiserver [flags]
 <td colspan="2">--authorization-config string</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>File with Authorization Configuration to configure the authorizer chain.Note: This feature is in Alpha since v1.29.--feature-gate=StructuredAuthorizationConfiguration=true feature flag needs to be set to true for enabling the functionality.This feature is mutually exclusive with the other --authorization-mode and --authorization-webhook-* flags.</p></td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>File with Authorization Configuration to configure the authorizer chain. This feature is mutually exclusive with the other --authorization-mode and --authorization-webhook-* flags.</p></td>
```

Comment:
This file is auto-generated. Your change will be overridden when v1.32 is out.
So ... there is no need to update this file.
If the flag description needs a change, the change should be proposed to the upstream source code in k/k, not here.

Reply:
reverting this since it has been addressed in https://github.com/kubernetes/kubernetes/commit/c383823228510e11bf63c07d910ed654efb99b67 and should be updated when 1.32 is released.